src/cryptsetup/cryptsetup-generator.c

```diff
@@ -110,7 +110,6 @@ static int create_disk(
                 "SourcePath=/etc/crypttab\n"
                 "DefaultDependencies=no\n"
                 "Conflicts=umount.target\n"
-                "BindsTo=dev-mapper-%i.device\n"
```

Comment:
Well, this PR is a workaround for #4725 

The root cause is https://github.com/systemd/systemd/commit/c5a97ed132b400ad82f7939d55fe1027a2b13f6e
> This adds a concept of GC'ing jobs of this type as soon as no client or
other job waits for them anymore

Seems like we can't reliable detect that. Does c5a97ed132b400 make sense at all?